db/migrate/20180810144738_update_default_internal_attribute.rb

```diff
@@ -0,0 +1,21 @@
+class UpdateDefaultInternalAttribute < ActiveRecord::Migration[5.0]
+  class ServiceTemplate < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled
+
+    include MigrationStubHelper
```

Comment:
I think this is only needed if you're using relations in the stub. Since you're only editing rows in the `service_templates` table I don't think this is needed.

@Fryguy @bdunne keep me honest here.